python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -1335,6 +1335,10 @@ def _get_proxy_base_class(cls):
     return object
 
 
+def isinstance_cudf_pandas(obj, type):
```

Comment:
My only question is, do we think it leaks too much implementation detail to rename this function to `is_proxy_instance` or similar? This isn't really pandas specific, and the fact that it's in `fast_slow_proxy.py` now makes that very explicit.

Reply:
You are right, I just renamed the check.